config/e3sm/machines/config_machines.xml

```diff
@@ -1147,10 +1147,11 @@
     <MAX_MPITASKS_PER_NODE>36</MAX_MPITASKS_PER_NODE>
     <PROJECT_REQUIRED>FALSE</PROJECT_REQUIRED>
     <mpirun mpilib="default">
-      <executable>srun</executable>
+      <executable>sbcast ${EXEROOT}/e3sm.exe /tmp/e3sm.exe; srun </executable>
       <arguments>
         <arg name="num_tasks"> -l -n {{ total_tasks }} </arg>
       </arguments>
+      <run_exe> /tmp/e3sm.exe </run_exe>
```

Comment:
Thanks, but please remove these changes to Anvil for now (keep `srun` for now). There are other unrelated issues. We'll put `sbcast ...` back when those are resolved.